webplatformcompat/view_serializers.py

```diff
@@ -21,7 +21,14 @@
 from .serializers import (
     BrowserSerializer, FieldMapMixin, FeatureSerializer, MaturitySerializer,
     SectionSerializer, SpecificationSerializer, SupportSerializer,
-    VersionSerializer, omit_some)
+    VersionSerializer, FieldsExtraMixin)
+
+
+def omit_some(source_list, *omitted):
+    """Return a list with some items omitted"""
+    for item in omitted:
+        assert item in source_list, '%r not in %r' % (item, source_list)
```

Comment:
Ahh... ok. Assuming I understand it right, that's essentially an excludes. If I recall, excludes-based things tend to be frowned on because it's too easy to accidentally make data public that shouldn't be public. I don't think that's a problem here except that one could easily accidentally change the API by adding new fields in the results.

If that's a big deal, then maybe add a test to make sure the fields you expect to be there are there? It'd fail if someone accidentally added a field thus nixing my concern without affecting yours. Something like this (pseudo-code):

```
def test_fields(self):
    serialized = ViewBrowserSerializer(some data)
    assert serialized.keys == expected_keys
```

I'm just thinking out loud. If it's not a big deal, then no biggie.